torch/csrc/distributed/c10d/CUDASymmetricMemory.cu

```diff
@@ -88,7 +88,9 @@ class IpcChannel {
     }
 
     TORCH_CHECK(
-        sendmsg(socket_, &msg, 0) > 0, "Failed to send fd: ", c10::utils::str_error(errno));
```

Comment:
Can you remove this file from the clang-format exclude list in lintrunner?

Reply:
Ah, it's missing from the include list: https://github.com/pytorch/pytorch/blob/070f3897453ea71a5505e4fcfaccb44e1e0cfa78/.lintrunner.toml#L56